sechub-wrapper-prepare/src/main/java/com/mercedesbenz/sechub/wrapper/prepare/prepare/PrepareWrapperContext.java

```diff
@@ -1,23 +1,47 @@
 package com.mercedesbenz.sechub.wrapper.prepare.prepare;
 
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
 import com.mercedesbenz.sechub.commons.model.SecHubConfigurationModel;
+import com.mercedesbenz.sechub.commons.model.SecHubMessage;
+import com.mercedesbenz.sechub.commons.model.SecHubRemoteDataConfiguration;
 import com.mercedesbenz.sechub.wrapper.prepare.cli.PrepareWrapperEnvironment;
 
 public class PrepareWrapperContext {
 
     private SecHubConfigurationModel secHubConfiguration;
     private PrepareWrapperEnvironment environment;
+    private List<SecHubRemoteDataConfiguration> remoteDataConfigurationList = new ArrayList<>();
+    private Collection<SecHubMessage> userMessages = new ArrayList<>();
```

Comment:
An array list is a collection, okay - a little bit irritating, but I would prefer here just
```suggestion
    private List<SecHubMessage> userMessages = new ArrayList<>();
```

Reason: It would be clear that it is ordered and could contain duplicates.